xbmc/pvr/PVREdl.cpp

```diff
@@ -9,7 +9,7 @@
 #include "PVREdl.h"
 
 #include "FileItem.h"
-#include "addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h"
+#include "addons/kodi-addon-dev-kit/include/kodi/addon-instance/PVR.h"
```

Comment:
If we would have separate header files for the c++ API classes, this would read `#include "addons/kodi-addon-dev-kit/include/kodi/addon-instance/PVR/EDL.h"`